modules/Material/IconButton.qml

```diff
@@ -54,12 +55,19 @@ Item {
         onClicked: {
             iconButton.clicked()
         }
+
+        onEntered: if (iconButton.hoverAnimation) icon.rotation = 90
```

Comment:
This can be simplified by adding the following directly to the Icon:

    rotation: ink.containsMouse ? 90 : 0